src/sentry/sentry_metrics/aggregation_option_registry.py

```diff
@@ -9,20 +9,27 @@ class AggregationOption(Enum):
     TEN_SECOND = "ten_second"
 
 
+class TimeWindow(Enum):
+    SEVEN_DAYS = "7d"
+    FOURTEEN_DAYS = "14d"
+    THIRTY_DAYS = "30d"
+    NINETY_DAYS = "90d"
+
+
 METRIC_ID_AGG_OPTION = {
-    "d:transactions/measurements.fcp@millisecond": AggregationOption.HIST,
-    "d:transactions/measurements.lcp@millisecond": AggregationOption.HIST,
+    "d:transactions/measurements.fcp@millisecond": {AggregationOption.HIST: TimeWindow.NINETY_DAYS},
```

Comment:
What do these `TimeWindow` represent when they are associated with an option?

In this dictionary, histogram is associated with 90 days, but I do not recall we set any sort of time parameters for the histogram aggregation in clickhouse. Does 90 days in this cas represent the entirety?

Reply:
This is the time window the option is available for. So 10 days is available for 7 days. I decided to explicitly say that we supported histograms for the entire 90 day window, just to avoid having to deal with defaults.